Bottom: b8889d4215ee1a9a5f62733eb8bbc283812c7e32
Top:    0688ce6af6e95d79135ff96c9e39fcc3c31084a8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 17:58:22 +0100

draw.c: Add prototype for via_callback

Fixes breakage since commit c095adf5ea890284cddc2730944b980ac1275109


---

diff --git a/src/draw.c b/src/draw.c
index 9f7cf8b..9f4a591 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -105,6 +105,7 @@ static void DrawHole (PinTypePtr);
 static void DrawMask (int side, BoxType *);
 static void DrawRats (BoxType *);
 static void DrawSilk (int, int, const BoxType *);
+static int via_callback (const BoxType * b, void *cl);
 static int pin_callback (const BoxType * b, void *cl);
 static int pad_callback (const BoxType * b, void *cl);
